src/psalm.php

```diff
@@ -110,7 +110,7 @@ function ($arg) use ($valid_long_options, $valid_short_options) {
                 );
                 exit(1);
             }
-        } elseif (substr($arg, 0, 2) === '-' && $arg !== '-' && $arg !== '--') {
+        } elseif (substr($arg, 0, 1) === '-' && $arg !== '-' && $arg !== '--') {
             $arg_name = preg_replace('/=.*$/', '', substr($arg, 1));
 
             if (!in_array($arg_name, $valid_short_options) && !in_array($arg_name . ':', $valid_short_options)) {
```

Comment:
The bug fix in this PR was noticed when looking at a check that `in_array()` is always explicitly passed a third argument for $strict.

It doesn't matter now, but might matter if `-0` becomes an option, because `'0' == '0e1'` (strings are cast to numbers to compare if both are numeric)